src/Routing/Filter/ThrottleFilter.php

```diff
@@ -53,13 +76,18 @@ protected function _initCache()
         }
     }
 
+    /**
+     * Atomically updates cache using default CakePHP increment offset 1.
+     *
+     * @param Cake\Network\Request $request Request instance
+     * @return Cake\Cache\Cache
+     */
     protected function _touch(Request $request)
     {
         $key = $this->config('identifier');
         if (is_callable($key)) {
             $key = $key($request);
         }
-
-        return Cache::increment($key, static::$cacheConfig);
+        return Cache::increment($key);
```

Comment:
I see the problem was wrong param value, second param is offset to increment. But don't just drop the cache config name. Pass it as 3rd param.